includes/footer.php

```diff
@@ -5,17 +5,83 @@
-                        <img class="img-responsive center-block" id="sponsor-boeing" alt="Boeing Logo" src="http://www.iptv.org/medialib/media/Boeing_RGB_blue_std_P.JPG"/>
+                <div id="carouselSponsor" class="carousel slide" data-ride="carousel" style="width: 400px; margin: 0 auto">
+                    <!-- Indicators -->
+                    <ol class="carousel-indicators">
+                        <li data-target="#carouselSponsor" data-slide-to="0" class="active"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="1"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="2"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="3"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="4"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="5"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="6"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="7"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="8"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="9"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="10"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="11"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="12"></li>
+                        
+                    </ol>
+
+                    <!-- Slides -->
+                    <div class="carousel-inner" role="listbox">
+                         <div class="item active">
+                            <img src="/img/sponsors/ehs.jpg" alt="/img/sponsors/ehs.jpg" title="Eastlake High School"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/google.jpg" alt="/img/sponsors/google.jpg" title="Google"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/lwsd.jpg" alt="/img/sponsors/lwsd.jpg" title="Lake Washington School District"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/boeing.jpg" alt="/img/sponsors/boeing.jpg" title="Boeing"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/chase.jpg" alt="/img/sponsors/chase.jpg" title="Chase"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/FIRSTWa.jpg" alt="/img/sponsors/FIRSTWa.jpg" title="FIRST Washington"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/godaddy.jpg" alt="/img/sponsors/godaddy.jpg" title="Go Daddy"/>
```

Comment:
It appears that GoDaddy is written without a space (based on a quick Google search). Change the `title` attribute to reflect this.